[ INFO ]- 开始加载数据库元数据
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 159, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 758, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 746, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\centralBrain.py", line 63, in init
File "E:/nvwa-run/trunk/loongtian/nvwa/centralBrainRuner.py", line 16, in <module> - 2019-03-26 15:48:43,966 - nvwa - logger.py:442
[ INFO ]- 加载数据库元数据完毕，没有可用的元数据
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 163, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 758, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 746, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\centralBrain.py", line 63, in init
File "E:/nvwa-run/trunk/loongtian/nvwa/centralBrainRuner.py", line 16, in <module> total_seconds:0.005 - 2019-03-26 15:48:43,970 - nvwa - logger.py:442
[ INFO ]- 开始从MetaNet数据库加载NgramDict
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 68, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 783, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 747, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\centralBrain.py", line 63, in init
File "E:/nvwa-run/trunk/loongtian/nvwa/centralBrainRuner.py", line 16, in <module> - 2019-03-26 15:48:43,971 - nvwa - logger.py:442
[ INFO ]- 从MetaNet数据库加载NgramDict完毕，没有可用的MetaNet
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 74, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 783, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 747, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\centralBrain.py", line 63, in init
File "E:/nvwa-run/trunk/loongtian/nvwa/centralBrainRuner.py", line 16, in <module> total_seconds:0.003 - 2019-03-26 15:48:43,974 - nvwa - logger.py:442
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1007, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 963, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-03-26 15:50:26,078 - nvwa - logger.py:442
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1007, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 963, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-03-26 15:51:39,595 - nvwa - logger.py:442
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1007, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 963, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-03-26 15:53:17,078 - nvwa - logger.py:442
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1007, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 963, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-03-26 15:54:02,634 - nvwa - logger.py:442
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1007, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 963, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-03-26 15:55:11,766 - nvwa - logger.py:442
[ INFO ]- “四头牛意义四头牛是集合四头牛元素牛四头牛数量四”思考出现错误，原因：需要匹配的模式（pattern）对象不是实际对象-占位符！
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\mind.py", line 109, in execute
File "E:\nvwa-run\trunk\loongtian\nvwa\centrals\thinkingCentral.py", line 63, in thinkStringInput
File "E:\nvwa-run\trunk\loongtian\nvwa\centrals\perceptionCentral.py", line 47, in receive
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\brain.py", line 99, in _execute
File "E:\nvwa-run\trunk\loongtian\util\tasks\runnable.py", line 45, in run - 2019-03-26 15:55:59,931 - nvwa - logger.py:442
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1007, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 963, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-03-26 15:56:18,125 - nvwa - logger.py:442
[ INFO ]- “四头牛意义四头牛是集合四头牛元素是牛四头牛数量四”思考出现错误，原因：需要匹配的模式（pattern）对象不是实际对象-占位符！
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\mind.py", line 109, in execute
File "E:\nvwa-run\trunk\loongtian\nvwa\centrals\thinkingCentral.py", line 63, in thinkStringInput
File "E:\nvwa-run\trunk\loongtian\nvwa\centrals\perceptionCentral.py", line 47, in receive
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\brain.py", line 99, in _execute
File "E:\nvwa-run\trunk\loongtian\util\tasks\runnable.py", line 45, in run - 2019-03-26 15:56:57,480 - nvwa - logger.py:442
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1007, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 963, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-03-26 15:58:09,960 - nvwa - logger.py:442
[ INFO ]- “五匹马意义五匹马是集合五匹马元素马五匹马数量五”思考出现错误，原因：参数错误！“意义”的右边meaning至少是个三元组，例如：牛-组件-腿。目前是：[{RealObject:{rid:e3d7210f4f9c11e9a2c740b89afe1ea2,remark:五}}]
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\mind.py", line 109, in execute
File "E:\nvwa-run\trunk\loongtian\nvwa\centrals\thinkingCentral.py", line 63, in thinkStringInput
File "E:\nvwa-run\trunk\loongtian\nvwa\centrals\perceptionCentral.py", line 47, in receive
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\brain.py", line 99, in _execute
File "E:\nvwa-run\trunk\loongtian\util\tasks\runnable.py", line 45, in run - 2019-03-26 15:58:12,926 - nvwa - logger.py:442
[ INFO ]- 开始加载数据库元数据
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 159, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 758, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 746, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\centralBrain.py", line 63, in init
File "E:/nvwa-run/trunk/loongtian/nvwa/centralBrainRuner.py", line 16, in <module> - 2019-03-28 10:54:54,365 - nvwa - logger.py:442
[ INFO ]- 加载数据库元数据完毕，没有可用的元数据
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 163, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 758, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 746, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\centralBrain.py", line 63, in init
File "E:/nvwa-run/trunk/loongtian/nvwa/centralBrainRuner.py", line 16, in <module> total_seconds:0.032 - 2019-03-28 10:54:54,398 - nvwa - logger.py:442
[ INFO ]- 开始从MetaNet数据库加载NgramDict
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 68, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 783, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 747, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\centralBrain.py", line 63, in init
File "E:/nvwa-run/trunk/loongtian/nvwa/centralBrainRuner.py", line 16, in <module> - 2019-03-28 10:54:54,398 - nvwa - logger.py:442
[ INFO ]- 从MetaNet数据库加载NgramDict完毕，没有可用的MetaNet
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 74, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 783, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 747, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\centralBrain.py", line 63, in init
File "E:/nvwa-run/trunk/loongtian/nvwa/centralBrainRuner.py", line 16, in <module> total_seconds:0.004 - 2019-03-28 10:54:54,403 - nvwa - logger.py:442
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1007, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 963, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-03-28 10:59:29,430 - nvwa - logger.py:442
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1007, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 963, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-03-28 11:04:19,494 - nvwa - logger.py:442
[ INFO ]- 开始加载数据库元数据
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 159, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 758, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 746, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\centralBrain.py", line 63, in init
File "E:/nvwa-run/trunk/loongtian/nvwa/centralBrainRuner.py", line 16, in <module> - 2019-04-08 15:03:04,404 - nvwa - logger.py:442
[ INFO ]- 加载数据库元数据完毕，没有可用的元数据
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 163, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 758, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 746, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\centralBrain.py", line 63, in init
File "E:/nvwa-run/trunk/loongtian/nvwa/centralBrainRuner.py", line 16, in <module> total_seconds:0.109 - 2019-04-08 15:03:04,513 - nvwa - logger.py:442
[ INFO ]- 开始从MetaNet数据库加载NgramDict
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 68, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 783, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 747, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\centralBrain.py", line 63, in init
File "E:/nvwa-run/trunk/loongtian/nvwa/centralBrainRuner.py", line 16, in <module> - 2019-04-08 15:03:04,513 - nvwa - logger.py:442
[ INFO ]- 从MetaNet数据库加载NgramDict完毕，没有可用的MetaNet
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 74, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 783, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 747, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\centralBrain.py", line 63, in init
File "E:/nvwa-run/trunk/loongtian/nvwa/centralBrainRuner.py", line 16, in <module> total_seconds:0.016 - 2019-04-08 15:03:04,529 - nvwa - logger.py:442
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1003, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 959, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-04-08 15:05:38,046 - nvwa - logger.py:442
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1003, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 959, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-04-08 15:07:59,990 - nvwa - logger.py:442
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1003, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 959, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-04-08 15:21:37,401 - nvwa - logger.py:442
[ INFO ]- 开始加载数据库元数据
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 159, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 758, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 746, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\centralBrain.py", line 63, in init
File "E:/nvwa-run/trunk/loongtian/nvwa/centralBrainRuner.py", line 16, in <module> - 2019-04-11 12:43:56,723 - nvwa - logger.py:442
